test/test_parser.cpp

```diff
@@ -4861,7 +4862,7 @@ TEST(Parser_TypeOfValue)
     verify_query(test_context, table, "mixed.@type == 'char'", nb_ints);
     verify_query(test_context, table, "mixed.@type == 'timestamp'", 0);
     verify_query(test_context, table, "mixed.@type == 'datetimeoffset'", 0);
-    verify_query(test_context, table, "mixed.@type == 'object'", 0);
+    verify_query(test_context, table, "mixed.@type == 'object'", 1);
```

Comment:
could you add a test for "objectlink" as well?

Reply:
Done